### PR TITLE
img attrs: srcset, currentSrc, sizes, referrerPolicy, decoding, loading

### PR DESCRIPTION
**Pull request by toastal**
### Prerequisites

- [x] Before opening a pull request, please check the HTML standard (https://dom.spec.whatwg.org/). If it doesn't appear in this spec, it may be present in the spec for one of the other `purescript-web` projects. Although MDN is a great resource, it is not a suitable reference for this project.

### Description

Adding missing attributes for `HTMLImage`
https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-currentsrc

It would be nice to include [`decode()`](https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-decode), but since it returns a `Promise` and this project doesn't include `Aff`, I skipped it.

### CODE REVIEW

File: src/Web/HTML/HTMLImageElement.purs [hunk 1]
```diff
@@ -101,6 +110,14 @@ foreign import setAlt :: String -> HTMLImageElement -> Effect Unit
 foreign import src :: HTMLImageElement -> Effect String
 foreign import setSrc :: String -> HTMLImageElement -> Effect Unit
 
+foreign import srcset :: HTMLImageElement -> Effect String
+foreign import setSrcset :: String -> HTMLImageElement -> Effect Unit
+
+foreign import currentSrc :: HTMLImageElement -> Effect String
+
+foreign import sizes :: HTMLImageElement -> Effect String
+foreign import setSizes :: String -> HTMLImageElement -> Effect Unit
```
Comment (garyb) [inline src/Web/HTML/HTMLImageElement.purs:130]: Missing an export for these

**Review by garyb — COMMENTED** [review]
Thanks for adding these, but can we introduce sum types for all the stringy attributes that only actually have a set of acceptable values? `CanPlayType` in this repo would be an existing example of that.

**Comment by toastal** [comment]
I wouldn't disagree, but it seemed like a lot of the APIs exposed in this repo are just stringly-typed. Can do though.

**Comment by toastal** [comment]
I think [`referrer-policy`](https://fetch.spec.whatwg.org/#concept-request-referrer-policy) may be out of scope as it refers to [specs](https://w3c.github.io/webappsec-referrer-policy/#referrer-policy-delivery-referrer-attribute) outside of HTML. Not really sure what's best @garyb. Though in going through the ADTs, I caught an error in `crossOrigin`.

File: src/Web/HTML/HTMLImageElement/Laziness.purs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    Lazy -> "Lazy"
+
+parse :: String -> Maybe Laziness
+parse = case _ of
+  "" -> Just Eager
+  "eager" -> Just Eager
+  "lazy" -> Just Lazy
+  _ -> Nothing
+
+
```
Comment (thomashoneyman) [inline src/Web/HTML/HTMLImageElement/Laziness.purs:29]:
```suggestion
```

File: src/Web/HTML/HTMLImageElement/Laziness.purs [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+print :: Laziness -> String
+print = case _ of
+  Eager -> "eager"
+  Lazy -> "lazy"
+
```
Comment (thomashoneyman) [inline src/Web/HTML/HTMLImageElement/Laziness.purs]:
```suggestion
```

**Review by thomashoneyman — APPROVED** [review]
This looks good to me, though I would appreciate another look from @garyb.

**Comment by garyb** [comment]
All seems reasonable to me! Thanks for adding those sums.

File: src/Web/HTML/HTMLImageElement/Laziness.purs (hunk 3 above)
Comment (toastal) [reply src/Web/HTML/HTMLImageElement/Laziness.purs]: `.editorconfig` :(

**Pull request merged**